File: Solved/10817/10817.cpp
// Problem No.: 10817
// Solver:      Jinmin Goh
// Date:        191120
// URL: https://www.acmicpc.net/problem/10817

#include <iostream>
#include <algorithm>
using namespace std;

int main()
{
    int a[3];
    cin >> a[0] >> a[1] >> a[2];
    sort(a, a + 3);
    cout << a[1];
    return 0;
}

File: Solved/10817/10817.py
# Problem No.: 10817
# Solver:      Jinmin Goh
# Date:        191120
# URL: https://www.acmicpc.net/problem/10817

a = input()
a = a.split()
for i in range(3):
    a[i] = int(a[i])

a.sort()
print(a[1])